src/app/tests/suites/certification/Test_TC_OO_2_7.yaml

```diff
@@ -0,0 +1,314 @@
+      cluster: "Groups"
+      command: "AddGroup"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+              - name: "GroupName"
+                value: "Group1"
+      response:
+          values:
+              - name: "Status"
+                value: 0
+              - name: "GroupID"
+                value: G1
+
+    - label:
+          "Step 1b: TH sends a RemoveAllScenes command to DUT with the GroupID
+          field set to G1."
+      command: "RemoveAllScenes"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+    - label:
+          "Step 1c: TH sends a GetSceneMembership command to DUT with the
+          GroupID field set to G1."
+      command: "GetSceneMembership"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+              - name: "SceneList"
+                value: []
+
+    - label: "Step 2a: TH sends Off command to DUT"
+      PICS: OO.S && !LVL.S.C04.Rsp
+      cluster: "On/Off"
+      command: "Off"
+
+    - label:
+          "Step 2a: TH sends a MoveToLevelWithOnOff command to DUT, with Level
+          =0 and TransitionTime =0 (immediate)"
+      PICS: OO.S && LVL.S.C04.Rsp
+      cluster: "Level Control"
+      command: "MoveToLevelWithOnOff"
+      arguments:
+          values:
+              - name: "Level"
+                value: 0
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
+
+    - label:
+          "Step 2b: TH sends a StoreScene command to DUT with the GroupID field
+          set to G1 and the SceneID field set to 0x01."
+      command: "StoreScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x01
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x01
+
+    - label: "Step 3: TH sends a AddScene command to DUT with the GroupID field
+          set to G1, the SceneID field set to 0x02, the TransitionTime field set
+          to 1000 (1s) and the ExtensionFieldSets set to: '[{ ClusterID: 0x0006,
+          AttributeValueList: [{ AttributeID: 0x0000, ValueUnsigned8: 0x01 }]}]'
+          "
+      command: "AddScene"
+      ## The default handlers for level control and on/off assume that if both clusters are in the same endpoint,
```

Comment:
Is that specified somewhere? If someone comes with a different interpretation, will they fail cert and is that as expected?

Reply:
That is how it is implemented in the SDK as off now, so I do not expect a different interpretation. 

That being said it isn't specified so I added a Spec PR here:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10114

We can have this as a followup, or should we leave the PR on hold?

Reply:
I wouldn't put it past matter.js to be ON it with scenes, btw. But the spec clarification works. As long as they both get it, let's not hold this up on that.